IMPLEMENTATION_COVERAGE.md

```diff
@@ -706,27 +706,37 @@
 
 ## bedrock
 <details>
-<summary>65% implemented</summary>
```

Comment:
These implementation coverage reports were all updated by the implementation_coverage.py script.